### DISCUSSION TITLE
What are the possible values for the `language` field in .nfo files?

### DISCUSSION THREAD

**Discussion by rodrigo-morales-1** [jellyfin/jellyfin#10907] [Q&A]
In [this page](https://jellyfin.org/docs/general/server/metadata/nfo) at the official documentation, the field `language` is listed under the section "Reading metadata from .nfo files". The section doesn't specify the domain for this field, so I'm not sure if I should write French or `fr` or some other acronym to represent the French language.

I want to add a `language` field to the metadata because I have media files in multiple languages and sometimes I want to filter media by language in order practice the selected language.

Currently, Jellyfin set the `language` field for audio streams, but I also want to set the `language` field for a media file. I'm planning to add this field when a movie has been produced in a single language (i.e. both the video stream and the original audio stream are in the same language).

**Comment by crobibero** [comment]
I don't see any sort of validation- but I may just be missing it. I would stick to ISO 639-1 codes that are supported by dotnet

https://www.loc.gov/standards/iso639-2/php/code_list.php
